benefits/eligibility/views.py

```diff
@@ -110,12 +110,19 @@ def start(request):
     )
 
     if verifier.is_auth_required:
-        if verifier.uses_auth_verification:
+        if verifier.auth_provider.claim == "senior":
```

Comment:
I think we should find a different way to do this. These values come out of secrets, so shouldn't be hardcoded and are configuration that could change in e.g. KeyVault and then this code would be broken.

Reply:
@machikoyasuda My thinking is that these bullets should be configured on the `EligibilityVerifier`, just like our many other msgids that are verifier-specific. The tricky thing here though is we have a variable number of bullet point items, i.e. a one-to-many relationship between verifier and bullet points. 

Django can of course support modeling this for us in the database, but it would mean that the bullet points have their own model/table! That seems like overkill.

Reply:
I think [#1442 ](https://github.com/cal-itp/benefits/issues/1442#issuecomment-1608572537) could be a potential solution here as well. Agency-specific tags that hardcodes the bullets for each.

Reply:
[`JSONField`](https://docs.djangoproject.com/en/4.2/ref/models/fields/#jsonfield) could be another way to just store a list of strings (the msgid)

Reply:
Going to try something like this:

1. Put all the strings into 1 msgid
2. Write a Django template filter that can take the stringified list and turn it back into a list, so `items.bullet` can still evaluate the list

Reply:
@thekaveman for #1442, how would we call Agency-specific or EligibilityVerifier-specific tags without using some kind of logic like `if verifier = x`?

Reply:
> @thekaveman for #1442, how would we call Agency-specific or EligibilityVerifier-specific tags without using some kind of logic like `if verifier = x`?

Yeah that is a good point @machikoyasuda we still need to make the decision somewhere. 

I think the idea of getting this data in the Verifier model is probably the way to go, as we've done for all other copy that varies per-flow.